src/ReleaseTests.package/ProperlyImplementedClassesTest.class/instance/testAllClassInstanceVariablesStartLowercase.st

```diff
@@ -0,0 +1,12 @@
+tests
+testAllClassInstanceVariablesStartLowercase
```

Comment:
This methods does not seems to belongs to this fix.

Reply:
Dont know how this came in. Maybe I added it to my development branch because this was from another PR that is already merged.

But it is the same version as the image already has so the change is safe to merge.